lib/hammer_cli_foreman/host.rb

```diff
@@ -173,14 +173,10 @@ def get_power_status
 
     class PuppetRunCommand < HammerCLIForeman::SingleResourceCommand
       command_name "puppetrun"
-      resource :puppet_hosts
-      action :puppetrun
 
-      def print_data(records)
-        print_message _('Puppet run triggered')
+      def execute
```

Comment:
Custom `execute` method's body must explicitly end with exit code.

I'd suggest `HammerCLI::EX_OK`. If you think that the command run should end with an error, you can use simple `1`.